microsite/data/plugins/gitlab-pipelines.yaml

```
@@ -13,4 +13,4 @@ tags:
   - gitlabCi
   - pipelines
   - jobs
-addedDate: '2023-09-27'
+addedDate: '2023-10-27'
```

```suggestion
addedDate: '2023-09-27'
```

Keeping this as is since that's when it was added